shared/constants/network.ts

```diff
@@ -818,7 +820,7 @@ export const CHAIN_ID_TO_RPC_URL_MAP = {
   [CHAIN_IDS.MEGAETH_TESTNET]: MEGAETH_TESTNET_RPC_URL,
 } as const;
 
-export const CHAIN_ID_TO_NETWORK_IMAGE_URL_MAP: Record<Hex, string> = {
+export const CHAIN_ID_TO_NETWORK_IMAGE_URL_MAP: Record<Hex | string, string> = {
```

Comment:
how awful would it be to keep this constant and create a new constant with a `Record<CaipChainId, string>` type that maps this existing evm `CHAIN_ID_TO_NETWORK_IMAGE_URL_MAP` values and adds the new solona one?

Reply:
i imagine we will want to do this eventually, but it's fine if that time is not now

Reply:
oh boy... I see we aren't the first tack this on :______) . Well then. I feel less guilt since other parties have already taken to bandaging